### DISCUSSION TITLE
File Manager: Column width

### DISCUSSION THREAD

**Discussion by nettsite** [webmin/authentic-theme#1659] [General]
How does one resize the column width in the File Manager?

![image](https://github.com/webmin/authentic-theme/assets/11553107/7c09cdcc-92b9-43b0-ba27-883726d4d886)

My "Name" column is very wide, and the rest are squashed. 

No double arrows appear if I scroll my mouse over the column boundaries.

**Comment by iliaross** [comment]
Hello,

This issue is already fixed in the latest version of Authentic theme.

**Reply by nettsite** [reply]
Gee, that was quick, thanks!

I am on version 20.21, do I need to upgrade?

**Comment by nettsite** [comment]
And the next question is how do I update it?

**Comment by nettsite** [comment]
OK, found it, upgraded, columns are spaced correctly. Thank you!

**Comment by iliaross** [comment]
Great to hear it.
